mirumee/saleor-dashboard
# Refactor of postalCodes related components

I want to merge this change because it refactors postal codes related components.
Clears up potential issues raised in https://github.com/mirumee/saleor-dashboard/pull/983
Testing:
- check shippingRatePrice update page
- check shippingRateWeight update page
- check shippingRatePrice create page
- check shippingRateWeight create page



**PR intended to be tested with API branch:** 

### Screenshots



### Pull Request Checklist



1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Translated strings are extracted.
1. [ ] Number of API calls is optimized.
1. [ ] The changes are tested.
1. [ ] Data-test are added for new elements.
1. [ ] Type definitions are up to date.
1. [ ] Changes are mentioned in the changelog.
1. [ ] The changes are tested in different browsers (Chrome, Firefox, Safari).
1. [ ] The changes are tested in light and dark mode.

### Test environment config



API_URI=https://master.staging.saleor.cloud/graphql/
